modules/muuntaja/project.clj

```diff
@@ -1,13 +1,18 @@
-(defproject metosin/muuntaja "0.6.6"
+(defproject metosin/muuntaja "0.0.0"                        ;; use lein v
```

Comment:
My opinion would be to just keep using `set-version` script.

Reply:
after giving it more thought, I agree on using the `set-version`